setup.py

```diff
@@ -197,7 +193,6 @@ def _setup_extras() -> Dict:
         "dev": _dev_deps,
         "server": _server_deps,
         "onnxruntime": _onnxruntime_deps,
-        "image_classification": _ic_integration_deps,
```

Comment:
do we need to pull out `deepsparse[image_classification]` out from any docs? the command might fail if we don't have this option

Reply:
Checked across the repo for references, we're good!